### PR TITLE
Ports and Network Interaction

### PR DESCRIPTION
**Pull request by askdba**
Signed-off-by: Alkin Tezuysal <alkin.tezuysal@gmail.com>

Documenting https://github.com/vitessio/website/issues/725

### CODE REVIEW

File: content/en/docs/user-guides/configuration-basic/ports.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Ports
+weight: 16
+aliases: ['/docs/launching/server-configuration/', '/docs/user-guides/server-configuration/', '/docs/user-guides/configuring-components/']
+---
+# Ports and Network interactions in Vitess
+
+Many/most of these ports are fully configurable, but we are listing their
```
Comment (hallaroo) [inline content/en/docs/user-guides/configuration-basic/ports.md:8]: Many/most of the ports listed below are fully configurable

File: content/en/docs/user-guides/configuration-basic/ports.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+defaults or by convention (e.g. from the examples) defaults here. Your
```
Comment (hallaroo) [inline content/en/docs/user-guides/configuration-basic/ports.md]: but we are listing their defaults or the defaults we use in examples here.

File: content/en/docs/user-guides/configuration-basic/ports.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+environment may differ considerably, depending on your configuration options
+for the various components:
+
+  * Data path - e.g.:
```
Comment (hallaroo) [inline content/en/docs/user-guides/configuration-basic/ports.md]: Don't think you need the e.g. here
Data path:

File: content/en/docs/user-guides/configuration-basic/ports.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    * Main query path:
+      * application -> vtgate
+        * port 3306 or 15306 (MySQL)
+        * port 15999 (gRPC)
+      * vtgate -> vttablet
+        * port 16000 + vttablet UID (gRPC)
+      * vttablet -> MySQL
+        * local socket (if MySQL is local)
+        * port 3306 (if MySQL is remote)
+    * vttablet -> vttablet:  vreplication within or across shards
```
Comment (hallaroo) [inline content/en/docs/user-guides/configuration-basic/ports.md]: To keep the formatting the same as the above

* vreplication within or across shards:
    * vttablet -> vttablet

File: content/en/docs/user-guides/configuration-basic/ports.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      * port 16000 + vttablet UID (gRPC)
+    * MySQL -> MySQL:  within-shard replication
```
Comment (hallaroo) [inline content/en/docs/user-guides/configuration-basic/ports.md]: To keep the formatting the same as the above

* within-shard replication:
    * MySQL -> MySQL

File: content/en/docs/user-guides/configuration-basic/ports.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      * port 3306 (MySQL protocol)
+  * Control or meta-data paths - e.g.:
```
Comment (hallaroo) [inline content/en/docs/user-guides/configuration-basic/ports.md]: Don't think you need the e.g.

* Control or meta-data paths:

**Review by hallaroo — CHANGES_REQUESTED** [review]
Just a few wording/formatting changes. Let me know when to re-review. Thanks!

**Comment by askdba** [comment]
@hallaroo added your comments to the doc.

**Review by hallaroo — APPROVED** [review]

**Pull request merged**